tests/XamlParserTests/Impl/Fixtures.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using XamlX;
+using Xunit;
+
+namespace XamlParserTests.Impl
+{
+
+    public class GuiLabsFixture : IDisposable
+    {
+        public GuiLabsFixture()
+        {
+            XamlParser.UseXDocumentParser = false;
```

Comment:
Probably we could simplify the code a bit if we just store the XAML parser (either `XDocument` or `GuiLabs`) under test in a field like `XamlParser` instead of dancing with a boolean flag.